pkg/platform/platform.go

```diff
@@ -252,3 +256,17 @@ func getMachineConfigPoolStatuses(ctx context.Context, client cnoclient.Client,
 	}
 	return mcpStatuses, nil
 }
+
+func consolePluginCRDExists(cl cnoclient.Client) bool {
+	consolePluginCrdKey := crclient.ObjectKey{Name: "consoleplugins.console.openshift.io"}
+	consolePluginCrdObj := &apiextensionsv1.CustomResourceDefinition{}
+	err := cl.Default().CRClient().Get(context.TODO(), consolePluginCrdKey, consolePluginCrdObj)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Printf("consoleplugins.console.openshift.io CRD was not found: %v", err)
+		} else {
+			log.Printf("ERROR: unable to get consoleplugins.console.openshift.io CRD: %v", err)
```

Comment:
The function should return this error, log it.
The error here should stop the bootstrap process.

Reply:
Done.
This function now returns an error in case the error is other than IsNotFound, and it is getting propagated into reconciliation error.